src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -589,6 +619,11 @@ impl ConstCx<'_, 'tcx> {
     }
 }
 
+/// Checks MIR for const-correctness, using `ConstCx`
```

Comment:
Const-correctness might confuse the hell out of someone from C / C++ ?

Reply:
Does it? Is that term used there for something? Do you have alternative suggestions?

Reply:
https://isocpp.org/wiki/faq/const-correctness#overview-const